src/Symfony/Bundle/FrameworkBundle/Resources/config/esi.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Resources\config;
```

Comment:
The namespace should be `namespace Symfony\Component\DependencyInjection\Loader\Configurator;`